tests/api/stats_cfg/test_stats_cfg_rest.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2023 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests REST API statistics configuration."""
+
+import json
+from copy import deepcopy
+
+from flask import url_for
+from invenio_accounts.testutils import login_user_via_session
+from utils import get_json, postdata
+
+
+def test_stats_cfg_permissions(client, stats_cfg_martigny, stats_cfg_sion,
```

Comment:
you can remove this test ; all permissions are already tested and "code covered" by specific permission test file.